assets/questions/i18n.js

```diff
@@ -87,8 +92,8 @@ def createListOfUniqueStrings(atom, size):
   }, {
     instructions: [
       [
-        "It's always important to think about the edge cases for your code. ",
-        'In this case, short strings are not abbreviated by this strategy. ',
+        "Next, consider possible edge cases for your code. One potential ",
+        'edge case is usually short strings. Does your code handle them properly? ',
         "Make sure your code doesn't try to abbreviate when it's not necessary."
```

Comment:
Actually, "necessary" here is part of the requirements and it actually does seem legitimate to me to represent "and" as "a1d" for consistency; I can also imagine another world where I wouldn't bother to abbreviate a 4-letter word.

So it might be worth being a little clearer here -- perhaps something like "... not necessary -- for example, "and" should remain as "and", but "beet" should change to "b2t".

This is just a suggestion, and other rewordings may be possible too.